docker-hub/billing/add-seats.md

```diff
@@ -4,45 +4,47 @@ keywords: Docker, Docker Hub, subscription, update, add, seats
 title: Add seats to your plan
 ---
 
-You can manage your Docker Hub subscription anytime by adding seats to your plan. If you add seats to your plan in the middle of your billing cycle, a prorated amount will be charged for the additional seats.
```

Comment:
> If you add seats to your plan in the middle of your billing cycle, a prorated amount will be charged for the additional seats.

Is that not true anymore?

Reply:
It's still true. The same info is in the notes so it was removed here
.